tests/test_ordinal_column.py

```diff
@@ -0,0 +1,161 @@
+"""
```

Comment:
Strings aren't supported as they have no strict ordering. Alphabetical is too weak an assumption to make.